app/app.go

```diff
@@ -230,6 +251,48 @@ func NewInitApp(
 		app.subspaces[slashing.ModuleName],
 	)
 
+	app.crisisKeeper = crisis.NewKeeper(
```

Comment:
This is useless if there is no route associated. Can be added later anyway, when/if we need to use the module.